13.Transform/assets/css/style.css

```diff
@@ -0,0 +1,264 @@
+:root {
```

Comment:
respect for using css variable but put this into separated file called variables